### PR TITLE
new ga changes by Ankit

### PR DESCRIPTION
**Pull request by ankitpanchal7**
Please take a pull and check.

### CODE REVIEW

File: workshop-butler/public/js/registration-page.js [hunk 1]
```diff
@@ -27,6 +27,8 @@ function register_attendee(e) {
             data: form_data,
             dataType: 'json'
         }).done(function(data) {
+            var wsb_ga_key = wsb_ga.google_analytics_key;
```
Comment (sery0ga) [inline workshop-butler/public/js/registration-page.js]: Please, use `let` instread of `var`

File: workshop-butler/public/js/registration-page.js [hunk 2]
```diff
@@ -35,6 +37,18 @@ function register_attendee(e) {
             jQuery('#wsb-success').show();
             $form.hide();
             $form.removeClass('.h-busy');
+
+            if( wsb_ga_key != '' ){
```
Comment (sery0ga) [inline workshop-butler/public/js/registration-page.js]: !==, not !=

File: workshop-butler/public/js/registration-page.js [hunk 3]
```diff
@@ -35,6 +37,18 @@ function register_attendee(e) {
...
+                (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```
Comment (sery0ga) [inline workshop-butler/public/js/registration-page.js]: This part requires a check. Google Analytics may be already on the page. So check 'ga' function for existance first before adding it.

**Comment by ankitpanchal7** [comment]
Hello,
Please check now.
I have fixed the issues.

Thanks.

**Pull request merged**